packages/ra-ui-materialui/src/input/AutocompleteInput.tsx

```diff
@@ -651,11 +651,7 @@ export const AutocompleteInputClasses = {
 const StyledAutocomplete = styled(Autocomplete, {
     name: PREFIX,
     overridesResolver: (props, styles) => styles.root,
-})(({ theme }) => ({
-    [`& .${AutocompleteInputClasses.textField}`]: {
-        minWidth: theme.spacing(20),
-    },
-}));
+})(() => ({}));
```

Comment:
I had to keep this empty function in order to not have the unit test breaking and kept the previous config line of the `styled` function call.

I am not very comfortable with the usage of the `styled` function. Is the `StyledAutocomplete` component still useful on that case?